### PR TITLE
Small addition to description of `head` command

### PR DESCRIPTION
**Pull request by abuts**
Resolves Re #1690. Added example of `head` command output. 
Not very useful, important or pretty but why not.

### CODE REVIEW

File: documentation/user_docs/docs/images/Head_output_table.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Scientific view:  ``head(file);`` ,Programmer view:  ``struc = head(file)``
```
Comment (cmarooney-stfc) [inline documentation/user_docs/docs/images/Head_output_table.csv:1]: Having looked at the file preview, it is very confusing as it stands. I keep thinking that there is a correlation between the items in each row, but there isn't. Could you break this into 2 tables, one with just the first column, one with just the second. Thanks

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

File: documentation/user_docs/docs/images/Head_output_table.csv (hunk 1 above)
Comment (abuts) [reply documentation/user_docs/docs/images/Head_output_table.csv:1]: There is some correlation between rows but you are probably right about confusion

File: documentation/user_docs/docs/manual/Special_sqw_information.rst [hunk 2]
```diff
@@ -48,6 +48,64 @@ an ``sqw`` or a ``dnd`` object. It also returns general information about the co
 this object, i.e. data ranges, number of pixels in ``sqw`` file etc., whatever developers decided 
 most important for user to know about correspondent object.
 
+Note, that output from calling ``head(filename);`` and ``res = head(filename)`` differs:
```
Comment (cmarooney-stfc) [inline documentation/user_docs/docs/manual/Special_sqw_information.rst:51]:
```suggestion
Note, that the outputs from calling ``head(filename);`` and ``res = head(filename)`` differ:
```
sorry, minor pedantry

File: documentation/user_docs/docs/manual/Special_sqw_information.rst [hunk 3]
```diff
@@ -48,6 +48,64 @@ an ``sqw`` or a ``dnd`` object. It also returns general information about the co
...
+
+>> ``head(filename);`` **"Scientific" view:**
```
Comment (cmarooney-stfc) [inline documentation/user_docs/docs/manual/Special_sqw_information.rst:53]: I don't understand why this one is called scientific and the next one is called programmers. They are different uses of the function; one gives printed output to the screen, one gives a variable (a struct) with similar but not identical information. A better description of the two views and a reason why they come out that way would be useful

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

File: documentation/user_docs/docs/manual/Special_sqw_information.rst (hunk 3 above)
Comment (abuts) [reply documentation/user_docs/docs/manual/Special_sqw_information.rst:53]: To be fair, I would throw away "scientifc" output and would leave only the second one. But I am certainly looking into it as a programmers.

But historically, this is what was displayed by Horace/disp and this constrans the information useful for a scientist. (not sufficient for any other purposes) I can not explain why this selection was done and do not like it but as it was probably reason behind it, I am maintaining it.

Comment (cmarooney-stfc) [reply documentation/user_docs/docs/manual/Special_sqw_information.rst:53]: ok if there are good historical reasons for this, let it stay

**Review by cmarooney-stfc — APPROVED** [review]

**Pull request merged**